src/config/constants.ts

```diff
@@ -53,6 +53,9 @@ export const GUARDIANS_URL = `${CLAIMING_DATA_URL}/guardians/guardians.json`
 export const GUARDIANS_IMAGE_URL = `${CLAIMING_DATA_URL}/guardians/images`
 export const VESTING_URL = `${CLAIMING_DATA_URL}/allocations`
 
+export const SEP5_EXPIRATION_DATE = '27.07.2023'
```

Comment:
Do we know the final date yet?

Reply:
Yes, this is the [announced date](https://5afe.slack.com/archives/C05EQB9HJQ5/p1688055490750249).

Reply:
Yeah, that's the date when it begins AFAIU. On your screenshot it says "until 27.07". Is it only one day?